services/frontend-service/src/ui/components/ApplicationLockDisplay/ApplicationLockDisplay.scss

```diff
@@ -0,0 +1,36 @@
+@use "@material/data-table/data-table";
```

Comment:
At least the CSS should be one for both components? or are there any differences here?

Reply:
No, now that only one component is used , it is more clear that only one use of data-table should be used